### ISSUE TITLE
App Debug build failed for pre lollipop with dynamic feature modules

### ISSUE THREAD

**Issue by burhankhanzada199888** [google/bundletool#157]
**Describe the bug**
when I run from app android studio its run gradle task `app:extarctApksForDebug` as I have dynamic features module project but on pre lollipop build failed 

**Bundletool version(s) affected**
I don't know what bundletool version it but I am using android gradle plugin version 3.6.3

**Stacktrace**
```
Program type already present: $r8$backportedMethods$utility$Integer$2$compare
Caused by: org.gradle.workers.internal.DefaultWorkerExecutor$WorkExecutionException: A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
Caused by: com.android.tools.build.bundletool.model.exceptions.CommandExecutionException: Dex merging failed.
Caused by: shadow.bundletool.com.android.tools.r8.CompilationFailedException: Compilation failed to complete
Caused by: shadow.bundletool.com.android.tools.r8.utils.AbortException: Error: Program type already present: $r8$backportedMethods$utility$Integer$2$compare
Error: Program type already present: $r8$backportedMethods$utility$Integer$2$compare
```

**To Reproduce**
well i don't know exactly how to reproduce that issue on your end but its only issue with pre lollipop build

**Expected behavior**
should debug build for pre lollipop device

**Environment:**
OS: Windows 10

**Comment by burhankhanzada199888** [comment]
Well this issue has with [https://github.com/ZieIony/Carbon](this) library I removed it and everything just works fine hope this helps anyone in future

**Comment by plecesne** [comment]
This is likely the same issue as https://issuetracker.google.com/issues/150400465

The fix went in AGP 4.0, so you can either join the beta while waiting it becomes stable, or you can use D8/R8 2.0.60 with AGP 3.6.3:

```
buildscript {
    repositories {
        maven {
            url 'https://storage.googleapis.com/r8-releases/raw'
        }
    }
    dependencies {
        classpath 'com.android.tools:r8:2.0.60'          // Must be before the Gradle Plugin for Android.
        classpath 'com.android.tools.build:gradle:3.6.3'
     }
}
```
